docs/devconsole/README.md

```diff
@@ -1,19 +1,38 @@
 # GitOps Setup on DevConsole
 
+Visualize your GitOps environment stages and applications on DevConsole in OpenShift 4.6
 
-1. Setup a GitOps pipeline on your cluster
-    * [setup up sample GitOps pipelines](./setup-gitops.md)
-2. Install GitOps service operator from Operator hub in all-namespaces.
-3. Go to DevConsole
-4. Application Stages nav-item should now be visible (feature-flagged on the availability of the operator)
-5. Create a namespace - 
+
+1. Follow the [Day 1 Operations](../journey/day1) and make sure your ArgoCD applications are in sync 
+
+![Applications](../journey/day1/img/ArgoCD_UI.png)
+
+2. Install the GitOps Service operator from OperatorHub in all-namespaces
+
+![Operator](./img/GitOps_Operator.jpg)
+
+![Installation](./img/GitOps_Installation.jpg)
+
+3. The operator will install the GitOps backend service in `openshift-pipelines-app-delivery` namespace. `Application Stages` nav-item should now be visible (feature-flagged on the availability of the operator)
```

Comment:
I'm not sure I'd say "feature-flagged" here, it's controlled by the availability of the operator.

Reply:
Resolved it here https://github.com/redhat-developer/kam/pull/60